Package.swift

```diff
@@ -7,6 +7,9 @@ let package = Package(
     name: "klaviyo-swift-sdk",
     platforms: [.iOS(.v13)],
     products: [
+        .library(
```

Comment:
SPM package changes to create new core module and establish the dependencies mentioned in the PR description.

Note to self - need to update cocoapods as well to mirror SPM changes.

Reply:
updated sdk to work with cocoapods.